examples/src/status_message.rs

```diff
@@ -18,14 +18,10 @@ async fn main() -> anyhow::Result<()> {
     println!("set_status: {:?}", outcome);
 
     let result: String = contract
-        .view(
-            "get_status",
-            json!({
-                "account_id": contract.id(),
-            })
-            .to_string()
-            .into_bytes(),
-        )
+        .view("get_status")
+        .args_json(json!({
```

Comment:
drive by, but some examples use `serde_json::json`, and some just `json`. I suggest unifying them all to `json`. Consistency is imporatnt for examples, because they get copy-pasted into downstream projects and set the initial patterns.